src/test/java/net/minecraftforge/PatchScreen.java

```diff
@@ -0,0 +1,17 @@
+package net.minecraftforge;
```

Comment:
Missing license header, please run the `updateLicenses`/`licenseFormat` task.

Additionally, move this to under the `.debug.client` subpackage please.